shuriken/lib/analysis/Dex/basic_blocks.cpp

```diff
@@ -137,26 +137,18 @@ shuriken::iterator_range<BasicBlocks::reverseedgesiterator_t> BasicBlocks::rever
 }
 
 shuriken::iterator_range<BasicBlocks::nodesetiterator_t> BasicBlocks::successors(DVMBasicBlock *node) {
-    if (successors_.find(node) == successors_.end())
-        throw std::runtime_error("Given node has no successors");
     return make_range(successors_[node].begin(), successors_[node].end());
```

Comment:
if `successors_` does not contain `node`, will calling `.begin()` raise a runtime exception? maybe we should keep the `if` but instead of throw, return an empty range. If the call will not crash, we can keep it as you wrote it.

Reply:
The change comes from encountering a run time error. After removing the throwing there's no crash anymore. I think at least the standard container, if the range is empty, the container will return .end() when begin() is called